### PR TITLE
Contact form branch Project

### PR DESCRIPTION
**Pull request by 3nityGitHub**
#In this PR,

- I created a new branch (Contact_form_branch)
- I added a new section/container for the contact form page
- I added the HTML form tags
- I styled the HTML form tags with CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -231,7 +232,7 @@ <h2 class="about_title">About me</h2>
             Get in touch
           </button>
         </form>
-       </section>
+    </section>
       <section class="footer_section">
```
Comment (MaganaAsiati) [inline index.html:236]: Kindly makes sure your icon design stick as much as possible to the Figma design provided as is shown in the picture below the color is not the same. To fix that I can suggest you use this  `color: #505F79` is the one used in the Figma design :+1: 

## You design 

image

## Figma design 

image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @3nityGitHub , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

- [ ] Kindly make sure the body takes a max screen so that it will remove the scrolling button from left to right as shown in the picture below. you can make that by giving the  body of your project the max of 100% :+1: 

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: styles.css [hunk 2]
```diff
@@ -44,12 +44,7 @@ header {
   flex-direction: column;
   justify-content: center;
   align-items: flex-start;
-  height: auto;
-}
-
-#sub_section2 {
-  margin-left: 20px;
-  color: #bac1fe;
+  height: 500px;
```
[the remark this answers is not in the document]
Comment (3nityGitHub) [reply styles.css:47]: @MaganaAsiati Thanks for the wonderful review and required changes. For this request, i honestly don't understand what you are talking about. Can you be more specific as to the changes you want me to make.
Thanks.

**Review by Qoosim — APPROVED** [review]
#  # *Status: Approved* :tada: :heavy_check_mark: 

Hi @3nityGitHub and @EngruugbeJimmy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_**Kindly check some optional comments I left under the review**_ :pray: 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [OPTIONAL] It is required for name input to have `maxlength="30"` not `maxlength="50"`. kindly change it.
- [OPTIONAL] As suggested by the previous reviewer. There's horizontal scroll bar on your page which is currently rendering your page with some faults. Take a look at some screenshots below and fix that.

hor-line-1


hor-line-2


hor-line-3


[OPTIONAL] This is stated optional, but highly recommended to fix it. One of the footer icons is now showing properly. Kindly look into that too and fix it.

your-footer-icon

**Pull request merged**